kpops/components/base_components/models/to_section.py

```diff
@@ -79,20 +79,18 @@ def extra_topic_role(cls, values):
 class ToSection(BaseModel):
     """Holds multiple output topics
 
-    :param models: Data models
-    :type models: dict[str, Any]
     :param topics: Output topics
-    :type topics: dict[str, TopicConfig]
+    :param models: Data models
     """
 
+    topics: dict[TopicName, TopicConfig] = Field(
```

Comment:
Add a snapshot test for this please 🧪

Reply:
there is no new behavior here, the only thing is that `ToSection` can be instantiated without arguments now, same as `FromSection`